examples/simple/posix/UB-PTHREAD6-bad.output

```diff
@@ -0,0 +1,7 @@
+Error: UB-PTHREAD6
+Description: Attempting to unlock a PTHREAD_MUTEX_DEFAULT or PTHREAD_MUTEX_NORMAL mutex which is not robust and which has not been locked by the current thread.
```

Comment:
@grosu  a robust mutex is a special type of mutex in POSIX 2008. Attempting to unlock a robust mutex does not result in undefined behavior, hence the error message. If you can think of another way of rephrasing this, I am open to suggestions.